### ISSUE TITLE
Learning rate increasing in YOLOv3

### ISSUE THREAD

**Issue by gnoya** [AlexeyAB/darknet#1737]
Hi, I'm currently training YOLOv3 with pre-trained darknet53.conv.74 weights. These are the cfg settings that I'm using:

![train_1](https://user-images.githubusercontent.com/22379971/46633372-c0101580-cb4d-11e8-9d2b-98e4df82eca3.png)

I started the training using ./darknet detector train ... ... ... -clear 1. After like 1000 iterations, the learning rate went from 0.000000 to 0.000001, and some iterations later went to 0.000002 (?). Then, before reaching 2000 iterations, went to 0.001000. That doesn't make any sense to me since the steps are 0.1.

Another question, 1 iteration means that the network trained for 1 batch or for whole training set once? Thanks!

**Comment by harsham05** [comment]
@AlexeyAB I have also noticed the ```learning_rate``` to increase while training, despite being initialized to 0.001 in the ```cfg``` file, & being reported to console output at the start of training.

Eg: from 0.000000 to 0.000002 , from start to 214 iterations.

1. params changed in ```cfg``` were
batch=64
subdivisions=16
classes=
filters=

**Comment by gnoya** [comment]
I have checked and found that the parameter burn_in increases the learning rate until burn_in iterations, and after that iteration the learning rate is the specified in the cfg file. Check this:

https://github.com/pjreddie/darknet/issues/701#issuecomment-382667417

**Comment by AlexeyAB** [comment]
About burn_in: https://github.com/AlexeyAB/darknet/issues/1161#issuecomment-403513728

burin_in - is used to try to find minimum the nearest to the pre-trained weights.

**Comment by theonsoong** [comment]
`1 iteration equals 1 batch.`@gnoya

**Comment by gnoya** [comment]
> About burn_in: [#1161 (comment)](https://github.com/AlexeyAB/darknet/issues/1161#issuecomment-403513728)
> 
> burin_in - is used to try to find minimum the nearest to the pre-trained weights.

Hi, I didn't get this, can you clarify it for me? Thanks!
